panels/templates/other_panelists_form.html

```diff
@@ -0,0 +1,49 @@
+<script type="text/javascript">
+    var showOrHideOtherPanelists = function () {
+        var count = $.val('other_panelists'),
+            $template = $('#panelist_form_template > .panel'),
+            $ops = $('#panelists_container > .panel');
+        setVisible('#panelists_container', count);
+        if (count < $ops.size()) {
+            $ops.slice(count).remove();
+        } else {
+            for (var i = $ops.size() + 1; i <= count; i++) {
+                var suffix = '_' + i;
+                var $opPanel = $template.clone();
+                $opPanel.find('h2').text($opPanel.find('h2').text() + i);
+                $opPanel.find('input').attr('name', function() { return $(this).attr('name') + suffix } );
+                $opPanel.find('textarea').attr('name', function() { return $(this).attr('name') + suffix } );
+                $opPanel.appendTo('#panelists_container');
+            }
+        }
+    };
+
+    $(function () {
+        showOrHideOtherPanelists();
+        $.field('other_panelists').on('change', showOrHideOtherPanelists);
+    });
+</script>
+
+<div class="form-group">
+  <div class="col-sm-6 col-sm-offset-3">
+    <select name="other_panelists">
+      {{ int_options(0, 9, other_panelists|length) }}
+    </select>
+    How many other panelists will be on this panel?
+  </div>
+</div>
+
+<div id="panelists_container">
+  {% for op in other_panelists %}
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <h2>Other Panelist #{{ loop.index }}</h2>
+        {{ panel_macros.panelist_form(
+            op,
+            suffix='_' + loop.index0|string,
```

Comment:
This loop may need to start at index 1. So you may need to change this to `suffix='_' + loop.index|string,`.

Reply:
I don't think this loop is ever actually executed, so we haven't encountered this bug yet. But I still think the suffix needs to start at index 1 instead of index 0.

If you check the definition of `compile_other_panelists_from_params()`, you'll notice that the range over which it iterates starts at 1: `for i in range(1, int(params.get('other_panelists', 0)) + 1):`

Reply:
Whoops, forgot to do that the first time!